platformcomponents/ios/toolbarButton.json

```diff
@@ -0,0 +1,17 @@
+{
+    "toolbarButton": {
+        "figma": "https://www.figma.com/file/L67wWWZaXcvVOLEpn77adE/Components---iOS?node-id=2741%3A6797",
+        "#normal": {
+            "background": "@theme-button-secondary-normal",
+            "text": "@theme-globaltint-normal",
+            "border": "@theme-global-tint-normal",
+            "badge": "@theme-presence-doNotDisturb"
+        },
+        "#pressed": {
+            "background": "@theme-globaltint-pressed"",
```

Comment:
There's an extra " at the end of this line. Please make sure you try building the tokens before submitting the PR.